From e9953f8f8b57261ebfc74c5474a77e47aaf6d35e Mon Sep 17 00:00:00 2001
From: Lubomir Rintel <lkundrak@v3.sk>
Date: Thu, 18 Dec 2014 14:45:37 +0100
Subject: [PATCH] write_page_ram: fix a typo

---
 main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/main.c b/main.c
index abce416..8638c2e 100644
--- a/main.c
+++ b/main.c
@@ -202,7 +202,7 @@ void write_page_ram(minipro_handle_t *handle, unsigned char *buf, unsigned int t
 	device_t *device = handle->device;
 	
 	int blocks_count = size / device->write_buffer_size;
-	if(size % device->read_buffer_size != 0) {
+	if(size % device->write_buffer_size != 0) {
 		blocks_count++;
 	}
 
